test/Microsoft.AspNetCore.Localization.FunctionalTests/TestRunner.cs

```diff
@@ -14,6 +14,8 @@ namespace Microsoft.AspNetCore.Localization.FunctionalTests
 {
     public class TestRunner
     {
+        private const string ApplicationBasePath = "http://localhost:0";
```

Comment:
Let's try to get to the bottom of this tomorrow and make sure we have a plan here.